### PR TITLE
Correct develop flow

### PR DESCRIPTION
**Pull request by gaborod16**
Align releases into develop.

### CODE REVIEW

**Comment by gaborod16** [comment]
It doesn't look like much, but this is actually aligning the Git History so new additions to the `master` branch avoid conflicts. :)

**Review by teddius — CHANGES_REQUESTED** [review]
This is not the desired development workflow since specifying the requirements in the setup.py is required for the installation dependency checks, hence this PR is rejected.

**Review by foldvaridominic — APPROVED** [review]

**Comment by gaborod16** [comment]
@teddius please read the code again.

The variable `requires` gathers all the information of the requirements from the `requirement.txt` dynamically, so you do not need to maintain duplicated information.

File: examples/agents/clean_polluted_annotations.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def get_all_entities(parent: str, language_code: str, client: Client) -> Dict[st
 
     for et in tqdm.tqdm(iterable=response.entity_types, desc='Loading entity map...'):
         for ev in et.entities:
-            _map[et.name][ev.name] = _map[et.name][ev.name].union({s for s in ev.synonyms})
+            _map[et.name][ev.name] |= {s for s in ev.synonyms}  # Merge sets together in place
```
Comment (kNalj) [inline examples/agents/clean_polluted_annotations.py:29]: Fancy operators.

**Review by kNalj — APPROVED** [review]
Looks good.

**Pull request merged**